docs/changes/v11.1.0/Tools-Changes.adoc

```diff
@@ -0,0 +1,15 @@
+= Tools Changes =
+
+== pkispawn and pkidestroy Changes ==
+
+Previously `pkispawn` and `pkidestroy` generated the logs on the console and also stored the logs into a file.
+The tools have been modified to only generate the logs on the console by default.
+To store the logs into a file as well, specify the file path using the `--log-file` option:
+
+----
+$ pkispawn ... --log-file <path>
+----
+
```

Comment:
I think this file should be moved into a `v11.0.3` folder since that would be the next version to be released from the `v11.0` branch. Also the "pkispawn and pkidestroy Changes" section should not be included since those changes were done in master (i.e. PKI 11.1) branch.

Reply:
Done.